{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/sqlalchemy_models/__init__.py

```diff
@@ -42,8 +42,35 @@ def get_tm_session(session_factory, transaction_manager):
           with transaction.manager:
               dbsession = get_tm_session(session_factory, transaction.manager)
 
+    This function may be invoked with a ``request`` kwarg, such as when invoked
+    by the reified ``request.dbsession`` Pyramid Request attribute. The default
+    value, for backwards compatibility, is ``None``.
+
+    - The ``request`` kwarg is used to populate the SQLAlchmey Session's "info"
+      dict.  The "info" dict is the official namespace for developers to stash
+      Session-specific information.  For more information, please see the
+      SQLAlchemy docs:
+      https://docs.sqlalchemy.org/en/13/orm/session_api.html?#sqlalchemy.orm.session.Session.params.info
+
+    - By placing the active Request in the "info" dict, developers will be able
```

Comment:
Why is this a list item and not just a plain old paragraph?

Also why is Request capitalized? It should either be lowercased to be a generic term of "request" or double-backticked as an inline literal.

Reply:
Will make both changes.  Request was capitalized to refer to the classname.